CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 [Full changelog](https://github.com/mozilla/glean/compare/v52.3.0...main)
 
+* Kotlin
+  * Allow the user to specify a custom data path when calling `initialize`.
```

Comment:
Let's clarify this a bit and instead of "when calling `initialize`", let's say we are adding the ability to record metrics from off the main process and link to the bug explaining the what and the why

Reply:
Fixed in https://github.com/mozilla/glean/pull/2433/commits/a0d09587cd56c122361ab8b948731b7902e96dc2